slave.mk

```diff
@@ -1386,6 +1386,7 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
         $(addprefix $(PYTHON_WHEELS_PATH)/,$(REDIS_DUMP_LOAD_PY2)) \
         $(addprefix $(PYTHON_WHEELS_PATH)/,$(SONIC_PLATFORM_API_PY2)) \
         $(addprefix $(PYTHON_WHEELS_PATH)/,$(SONIC_PLATFORM_API_PY3)) \
+        $(if $(findstring mellanox,$(CONFIGURED_PLATFORM)),$(addprefix $(PYTHON_WHEELS_PATH)/,$(SONIC_PLATFORM_API_DPUCTL_PY3))) \
```

Comment:
is this required? there is an install in the sonic_debian_extention

Reply:
If this line is removed the wheel file is not generated. This update generates the wheel file and the update in sonic_debian_extension copies the wheel file and installs it using pip. This is the way the other wheel files are installed as well, If theres a way to only update the mellanox section in sonic_debian_extension that would be better, but I could not find such references

Reply:
I see. I also could not find a better way. As a side note, we might want to consider adding the possibility to have a platform-specific python wheels (e.g $(SONIC_ONE_IMAGE)_PYTHON_WHEELS)